gobblin-modules/gobblin-sql/src/main/java/org/apache/gobblin/writer/commands/MySqlWriterCommands.java

```diff
@@ -145,7 +149,13 @@ public void drop(String database, String table) throws SQLException {
 
   @Override
   public void copyTable(String databaseName, String from, String to) throws SQLException {
-    String sql = String.format(COPY_INSERT_STATEMENT_FORMAT, databaseName, to, databaseName, from);
+    // Chooses between INSERT and REPLACE logic based on the job configurations
+    String sql;
+    if (this.replaceExistingValues) {
+      sql = String.format(COPY_REPLACE_STATEMENT_FORMAT, databaseName, to, databaseName, from);
+    } else {
+      sql = String.format(COPY_INSERT_STATEMENT_FORMAT, databaseName, to, databaseName, from);
+    }
```

Comment:
minor: the ternary op `c ? x : y` is nice for avoiding the need to initialize empty.  here it could even be tucked within the first arg expr to `String.format`.